cloudflare/resource_cloudflare_spectrum_app.go

```diff
@@ -0,0 +1,313 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/pkg/errors"
+)
+
+const (
+	cloudflareOn  = "on"
+	cloudflareOff = "off"
+)
+
+func resourceCloudflareSpectrumApp() *schema.Resource {
```

Comment:
In relation to my previous comment, we probably would want to `s/app/application` for all of these functions.

Reply:
yep :+1: